<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportWizardWidget</class>
 <widget class="QWidget" name="ReportWizardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReportWizardWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="deviceTypePage">
      <layout class="QGridLayout" name="gridLayout_13">
       <item row="0" column="0">
        <widget class="QGroupBox" name="deviceTypesGroupBox">
         <property name="title">
          <string>选择设备类型</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="2" column="0">
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="1" column="1">
              <widget class="QCheckBox" name="killTimeCheckBox">
               <property name="text">
                <string>测量时间</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QCheckBox" name="maxCheckBox">
               <property name="text">
                <string>最 大 值</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="timeZoneCheckBox">
               <property name="text">
                <string>灭菌时段</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QCheckBox" name="averageCheckBox">
               <property name="text">
                <string>测量均值</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="minCheckBox">
               <property name="text">
                <string>最 小 值</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QComboBox" name="deviceTypeComboBox">
               <property name="currentIndex">
                <number>1</number>
               </property>
               <item>
                <property name="text">
                 <string>环境试验设备</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>蒸汽灭菌设备</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>干热灭菌设备</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>清洗消毒设备</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>微波消解设备</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QCheckBox" name="valueA0CheckBox">
               <property name="text">
                <string>测量 A0 值</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QCheckBox" name="valueF0CheckBox">
               <property name="text">
                <string>测量 F0 值</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QCheckBox" name="bodongCheckBox">
               <property name="text">
                <string>波 动 度</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>1</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="commonInformationPage">
      <layout class="QGridLayout" name="gridLayout_15">
       <item row="0" column="0">
        <widget class="QGroupBox" name="commonInformationGroupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>通用信息</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>用户相关信息 (会生成在报告中)</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>公司名称:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="companyNameLineEdit"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>部门名称:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="departmentLineEdit"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>详细地址:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="addressLineEdit"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>电话号码:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="phoneNumberLineEdit"/>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>作业专家:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="expertLineEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="1" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>设备名称:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>厂家相关信息 (会生成在报告中)</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="deviceProviderLineEdit"/>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="deviceModelLineEdit"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>设备编号:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>设备型号:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="deviceSerialLineEdit"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>设备厂家:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="deviceNameLineEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>1</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="detailParametersPage">
      <layout class="QGridLayout" name="gridLayout_14">
       <item row="0" column="0">
        <widget class="QGroupBox" name="detailParametersGroupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>灭菌参数</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="1" column="0">
           <widget class="QFrame" name="frame_6">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_10">
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>灭菌平台判定规则</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>毛刺处理:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="curvePropertyComboBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <item>
                <property name="text">
                 <string>曲线平滑禁止</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>曲线平滑使能</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>压力辅助:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="curvePressComboBox">
               <item>
                <property name="text">
                 <string>灭菌平台结束判断 - 饱和蒸汽辅助禁用</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>灭菌平台结束判断 - 饱和蒸汽辅助使能</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QFrame" name="mieJunParametersFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="1" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>灭菌温度:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="killTempLineEdit"/>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>摄氏度</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>灭菌时间:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="killTimeLineEdit"/>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>秒 (S)</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="3">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>灭菌参数设置 (环境试验设备不用关心)</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QFrame" name="frame_7">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_11">
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>记录器布点图</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>布点开关:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="buDianComboBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <item>
                <property name="text">
                 <string>分析报告附加点布点图禁止</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>分析报告附加点布点图使能</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>重新布点:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="buDianButton">
               <property name="text">
                <string>打开布点图开始布点</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>1</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="reportParametersPage">
      <layout class="QGridLayout" name="gridLayout_17">
       <item row="0" column="0">
        <widget class="QGroupBox" name="reportParametersGroupBox">
         <property name="title">
          <string>选择报告参数</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QFrame" name="frame_8">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_16">
             <item row="0" column="1">
              <widget class="QCheckBox" name="report2CheckBox">
               <property name="text">
                <string>温度 F0 值报告</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="report3CheckBox">
               <property name="text">
                <string>综合分析报告</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="report4CheckBox">
               <property name="text">
                <string>审计追踪报告</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="report1CheckBox">
               <property name="text">
                <string>统计表格报告</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>312</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="previousButton">
       <property name="text">
        <string>上一步</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>下一步</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>deviceTypeComboBox</tabstop>
  <tabstop>timeZoneCheckBox</tabstop>
  <tabstop>killTimeCheckBox</tabstop>
  <tabstop>averageCheckBox</tabstop>
  <tabstop>maxCheckBox</tabstop>
  <tabstop>minCheckBox</tabstop>
  <tabstop>bodongCheckBox</tabstop>
  <tabstop>valueF0CheckBox</tabstop>
  <tabstop>valueA0CheckBox</tabstop>
  <tabstop>companyNameLineEdit</tabstop>
  <tabstop>departmentLineEdit</tabstop>
  <tabstop>addressLineEdit</tabstop>
  <tabstop>phoneNumberLineEdit</tabstop>
  <tabstop>expertLineEdit</tabstop>
  <tabstop>deviceNameLineEdit</tabstop>
  <tabstop>deviceModelLineEdit</tabstop>
  <tabstop>deviceSerialLineEdit</tabstop>
  <tabstop>deviceProviderLineEdit</tabstop>
  <tabstop>killTempLineEdit</tabstop>
  <tabstop>killTimeLineEdit</tabstop>
  <tabstop>curvePropertyComboBox</tabstop>
  <tabstop>curvePressComboBox</tabstop>
  <tabstop>buDianComboBox</tabstop>
  <tabstop>buDianButton</tabstop>
  <tabstop>report1CheckBox</tabstop>
  <tabstop>report2CheckBox</tabstop>
  <tabstop>report3CheckBox</tabstop>
  <tabstop>report4CheckBox</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>previousButton</tabstop>
  <tabstop>nextButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
